scripts/filter_multimappers.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+
+import argparse
+import pysam
+import sys
+
+def parse_arguments():
+    '''
+    Command-line arguments parser
+    '''
+    parser = argparse.ArgumentParser(
+        description="Script to extend pre-miRNs overhang."
+        )
+
+    parser.add_argument(
+        '-v','--version', 
+        action='version', 
+        version='%(prog)s 1.0',
+        help="Show program's version number and exit"
+        )
+
+    parser.add_argument(
+        '-i', '--input', 
+        help="Path to the gff3 input file",
```

Comment:
Sure it's gff3? I think it's SAM, no?

"Path to input file in SAM format, sorted by query name"